spring-cloud-data-shell/src/test/java/org/springframework/cloud/data/shell/AbstractShellIntegrationTest.java

```diff
@@ -24,14 +24,14 @@
 import org.junit.rules.TestName;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
```

Comment:
preserve this newline... updated your organize imports prefs to treat "org.springframework" as a specific grouping